script.js

```diff
@@ -0,0 +1,11 @@
+let input = prompt('Please type in a number between 1 and 100.');
+
+let divide = input / 2;
+let multiply = input * 2;
+
+let msg = "Your number is: " + input; 
+console.log(msg);
+msg = "Half of your number is: " + divide;
+console.log(msg);
+msg = "Double your number is: " + multiply;
```

Comment:
```suggestion
msg = "Double your number is: " + multiply;
document.write(msg);
```
You are just missing the requirement to put the messages on the page with document.write